### DISCUSSION TITLE
Start eyery second dbus service - no module named yaml

### DISCUSSION THREAD

**Discussion by Walkinlarge2010** [freakent/dbus-mqtt-devices#115] [Q&A] [answered]
Hi everybody,

i have a problem, wich i dont know how to solve it an i want not to destroy my cerbo gx ;) 

i have the problem, that every to days my system have failures. So i made an FW upgrade on my cerbo gx to 3.30. After restart i couldnt connect. 

If i check the dbus service is running, ill get the wollowing answer:
`root@einstein:~# svstat /service/dbus-mqtt-devices
/service/dbus-mqtt-devices: up (pid 26870) 1 seconds
root@einstein:~# svstat /service/dbus-mqtt-devices
/service/dbus-mqtt-devices: up (pid 26911) 0 seconds
root@einstein:~# svstat /service/dbus-mqtt-devices
/service/dbus-mqtt-devices: up (pid 26934) 1 seconds`

you can see, it es every secon a new pid. it seems like there is an problem

in the log i saw the following message:

`@4000000065fff75f090522ac ModuleNotFoundError: No module named 'yaml'
@4000000065fff75f1253a0a4 *** starting dbus-mqtt-devices ***
@4000000065fff7602ce343ac Traceback (most recent call last):
@4000000065fff7602ce366d4   File "/data/drivers/dbus-mqtt-devices-0.6.6/dbus_mqtt_devices.py", line 34, in <module>
@4000000065fff7602ce37674     from device_manager import MQTTDeviceManager
@4000000065fff7602ce3822c   File "/data/drivers/dbus-mqtt-devices-0.6.6/device_manager.py", line 17, in <module>
@4000000065fff7602ce38de4     import yaml
@4000000065fff7602ce395b4 ModuleNotFoundError: No module named 'yaml'
@4000000065fff76035fba9fc *** starting dbus-mqtt-devices ***
@4000000065fff76218ea884c Traceback (most recent call last):
@4000000065fff76218eaa78c   File "/data/drivers/dbus-mqtt-devices-0.6.6/dbus_mqtt_devices.py", line 34, in <module>
@4000000065fff76218eab72c     from device_manager import MQTTDeviceManager
@4000000065fff76218eac2e4   File "/data/drivers/dbus-mqtt-devices-0.6.6/device_manager.py", line 17, in <module>
@4000000065fff76218eace9c     import yaml
@4000000065fff76218ead66c ModuleNotFoundError: No module named 'yaml'`

google said there is a problem with python, now iam not sure, whats the best way to fix this.

Has anyone have an advice for me? In the worst case ill make an hard reset on my cerbo an install everything new, but this shouldt be the last option.

Thanks

Best regard Matthias

**Comment by freakent** [comment, marked as answer]
Simple fix, just run the ./bin/setup.sh again. 

Next release it will run the required steps from the setup.sh automatically.

**Reply by Walkinlarge2010** [reply]
Thanks, now it works again 👌
